### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by scruffian** [tzachz/github-comment-counter#8]
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':leaderboard-server:runtimeCopy'.
> Cannot resolve external dependency com.google.guava:guava:14.0.1 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency joda-time:joda-time:2.2 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency org.slf4j:slf4j-api:1.7.5 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency com.yammer.dropwizard:dropwizard-core:0.6.2 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef
> Cannot resolve external dependency com.yammer.dropwizard:dropwizard-views:0.6.2 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef
> Cannot resolve external dependency com.sun.jersey:jersey-client:1.17.1 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency com.sun.jersey:jersey-core:1.17.1 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency com.fasterxml.jackson.core:jackson-databind:2.1.4 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.1.4 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef
> Cannot resolve external dependency ch.qos.logback:logback-classic:1.0.10 because no repositories are defined.
  Required by:
      com.tzachz:leaderboard-server:0.1.undef > com.tzachz:github-comment-fetcher:0.1.undef

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED```

**Comment by litalhassine** [comment]
Try adding the + lines in build.gradle (that what worked for me):

 allprojects {
     apply plugin: 'java'
     apply plugin: 'idea'
 
+    repositories {
+        mavenCentral()
+    }
+
     project.group = 'com.tzachz'
     project.version = "0.1.$BUILD_NUMBER"

 as part of allprojects block

**Comment by tzachz** [comment]
Thanks @litalhassine - that would work. I will update master to include this.
Alternatively - I use a local `~/.gradle/gradle.init` that points to all the repositories I use, so that repositories are per environment and not per project. 
A basic .init file would contain:
```
allprojects {
    repositories {
        mavenLocal()
        mavenCentral()
    }
    buildscript {
        repositories {
            mavenLocal()
            mavenCentral()
        }
    }
}

```
See more info here: https://gradle.org/docs/current/userguide/init_scripts.html

**Comment by tzachz** [comment]
Fixed by #9. Thanks for the report.
